simulator/mips/mips_instr.h

```diff
@@ -332,14 +331,14 @@ class MIPSInstr
 
         const std::string_view Dump() const { return static_cast<std::string_view>(disasm); }
         bool is_same( const MIPSInstr& rhs) const {
-            return PC == rhs.PC && instr.raw == rhs.instr.raw;
+            return bp_update.branch_ip == rhs.bp_update.branch_ip && instr.raw == rhs.instr.raw;
```

Comment:
For instance, it become hardly readable this method just compares that instructions were read from the same address